### DISCUSSION TITLE
How to choose an atmospheric model for fitting contact binaries?

### DISCUSSION THREAD

**Discussion by dingxu6207** [phoebe-project/phoebe2#799] [Questions]
Can I choose a blackbody atmosphere instead of a model atmosphere for contact binary?  What atmospheric model should be chosen for semi-detached binaries and detached binaries?

**Comment by aprsa** [comment]
Unless your temperatures are out of model atmosphere bounds, you should never use blackbody atmospheres. The upcoming version of phoebe will support automated blending but we're not quite there yet. As for the choice of a model atmosphere, I would go with either ck2004 or phoenix, and possibly both to compare, depending on the temperature. For massive contact binaries @MichaelAbdul-Masih developed a phoebe-based spamms code that might also fit your needs. Contact binaries are most susceptible to atmospheric effects so I wouldn't contemplate blackbody if I didn't need to.

**Reply by dingxu6207** [reply]
Many Thinks!
